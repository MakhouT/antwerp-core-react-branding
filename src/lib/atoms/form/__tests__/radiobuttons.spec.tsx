import * as React from "react";

import * as A from "../../..";
import { renderAndMatchSnapshot } from "../../../../__tests__/__fixtures__/testUtils";

describe("Radiobuttons", () => {
    it("Renders default", () => {
        // Arrange
        const element = <A.Radiobuttons
            name="radiobuttons"
            options={
                [
                    { label: "a", value: "a" },
                    { label: "b", value: "b" },
                    { label: "c", value: "c" },
                ]
            }

        />;

        // Act and assert
        renderAndMatchSnapshot(element);
    });
});
