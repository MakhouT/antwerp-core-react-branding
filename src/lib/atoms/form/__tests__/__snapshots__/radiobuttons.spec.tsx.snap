// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Radiobuttons Renders default 1`] = `
<div
  className="a-input"
>
  <div
    className="a-input__radio"
  >
    <input
      checked={false}
      disabled={undefined}
      id="radiobutton-radiobuttons-0"
      name="radiobuttons"
      onChange={[Function]}
      type="radio"
      value="a"
    />
    <label
      className=""
      htmlFor="radiobutton-radiobuttons-0"
    >
      a
    </label>
  </div>
  <div
    className="a-input__radio"
  >
    <input
      checked={false}
      disabled={undefined}
      id="radiobutton-radiobuttons-1"
      name="radiobuttons"
      onChange={[Function]}
      type="radio"
      value="b"
    />
    <label
      className=""
      htmlFor="radiobutton-radiobuttons-1"
    >
      b
    </label>
  </div>
  <div
    className="a-input__radio"
  >
    <input
      checked={false}
      disabled={undefined}
      id="radiobutton-radiobuttons-2"
      name="radiobuttons"
      onChange={[Function]}
      type="radio"
      value="c"
    />
    <label
      className=""
      htmlFor="radiobutton-radiobuttons-2"
    >
      c
    </label>
  </div>
  <small
    className=""
  >
     
  </small>
</div>
`;
